languages/c/reference/exercise-concepts/two-fer.md

```diff
@@ -0,0 +1,22 @@
+# Concepts of two-fer
+
+```c
+#include "two_fer.h"
+#include <stdio.h>
+
+void two_fer(char *buffer, const char *name) {
+    sprintf(buffer, "One for %s, one for me.", name ? name : "you");
+}
+```
+
+- [Header Files](https://www.gnu.org/software/libc/manual/html_mono/libc.html#Header-Files): used to access the standard library
```

Comment:
Something's gone wrong with the markdown rendering here - maybe its GitHub's renderer, or maybe the syntax is wrong?

Reply:
I think it's GitHub's commit renderer I was viewing it in, it looks fine in md file preview view 😄